prymer/offtarget/offtarget_detector.py

```diff
@@ -11,7 +11,7 @@
 
 ```python
 >>> from pathlib import Path
->>> from prymer.api.span import Strand
+>>> from prymer.model import Strand
```

Comment:
I think in the docs we should import these as we would recommend users to import them, namely from the top-level prymer namespace.  If you agree, can you fix in the rest of the modules as well?

```suggestion
>>> from prymer import Strand
```